README.md

```diff
@@ -83,6 +83,9 @@ No, the plugin provides an easy-to-use interface where you can add schema markup
 
 ## Changelog ##
 
+### 1.7.2 ###
+- Improvement: Integrated PHP Insights for automated code quality analysis.
```

Comment:
- Improvement: Enhanced code quality and overall plugin performanc


there is no need to inform users about PHPInsights